packages/nest/README.md

```diff
@@ -61,6 +61,19 @@ Capabilities include:
 npm install --save @openfeature/nestjs-sdk
 ```
 
+#### npm
```

Comment:
README change just to prompt a new release, but also we were missing the yarn section.